scripts/run

```diff
@@ -38,4 +38,23 @@ if [ -f /sys/fs/cgroup/cgroup.controllers ]; then
   sed -e 's/ / +/g' -e 's/^/+/' <"/sys/fs/cgroup/cgroup.controllers" >"/sys/fs/cgroup/cgroup.subtree_control"
 fi
 
+if [ $CI ]; then
+  # Attempt to find and stop any previous instances of dapper containers.
+  # To combat the scenario where drone runners are reused and an instance of rancher is still running.
+  echo "Checking for running containers from a previous build."
+
+  # gets a list of all docker containers that were started in a CI build.
+  dapper_containers=$(docker ps --format '{{.Names}}/{{.Label "DRONE_BUILD_NUMBER"}}' --filter 'label=CI=true')
```

Comment:
The current autoscaler configurations have the [`DRONE_AGENT_CONCURRENCY`](https://autoscale.drone.io/reference/drone-agent-concurrency/) value set to "1" for all platforms. So yes, while a node may be reused **across** builds (workers persist as long as there are pending jobs, or until the reach [`DRONE_POOL_MIN_AGE`](https://autoscale.drone.io/reference/drone-pool-min-age/), which is currently set to `5m` for Linux nodes and `35m` for Windows nodes), it will not run multiple builds concurrently.

It's probably worth adding a comment to the code indicating that the safety of this fix is dependent on that configuration.